scapy/layers/l2.py

```diff
@@ -451,6 +452,34 @@ class ERSPAN(Packet):
                    ]
 
 
+class ERSPAN_III(Packet):
```

Comment:
I think that we should move the headers to `contrib/` as they are not common on a network.